omnibus/resources/chef-workstation/pkg/welcome.html.erb

```diff
@@ -5,14 +5,18 @@
 </head>
 <body>
 <font face="Helvetica">
-This package will install <b><%= friendly_name %></b>, which has everything you need to get started with Chef.
+This package will install <b><%= friendly_name %></b>, which has everything you need to get started with Chef products.
 <br>
 <br>
 Including:
 <ul>
-  <li><strong>chef-run:</strong> Ad-hoc remote execution of Chef</li>
```

Comment:
@tas50 we don't want to mantion the `chef-run` tool?

Reply:
I added it back